test/core/client_channel/lb_policy/lb_policy_test_lib.h

```diff
@@ -538,6 +543,41 @@ class LoadBalancingPolicyTest : public ::testing::Test {
     return retval;
   }
 
+  // Waits for the round_robin policy to start using an updated address list.
+  // There can be any number of READY updates where the picker is still using
+  // the old list followed by one READY update where the picker is using the
+  // new list.  Returns true if the reported states match expectations.
+  bool WaitForRoundRobinListChange(
```

Comment:
I suggest moving RR-specific utilities to a separate file. I am concerned that current framework actively encourages using RR for testing, e.g., host override policy. I am certain this will result in test coverage gaps...

Reply:
I don't see why using RR for testing will result in test coverage gaps.  RR is really the most basic policy we have, and I think in the vast majority of cases, it's going to be the best choice to use as a child policy when we're testing a non-leaf policy.  The reason I put this code here is that I actively *want* to encourage using it, and I think methods like this will make it easier to do that in a way that verifies the picker behavior after every change.